evalai/utils/requests.py

```diff
@@ -90,19 +62,12 @@ def make_request(path, method, files=None, data=None):
         echo(
             style(
                 "You can use `evalai submission {}` to view this submission's status.\n".format(
-                    response.get("id")
+                    response.json().get("id")
                 ),
                 bold=True,
-                fg="white"
             )
         )
-        return response
-    elif method == "PUT":
-        # TODO: Add support for PUT request
-        pass
-    elif method == "PATCH":
-        # TODO: Add support for PATCH request
-        pass
-    elif method == "DELETE":
-        # TODO: Add support for DELETE request
-        pass
+    # TODO: Add support for PUT request
+    # TODO: Add support for PATCH request
+    # TODO: Add support for DELETE request
```

Comment:
Please remove these TODOs as the `requests.request` will implement this as well

Reply:
Okay. That's done! :D :+1: